source/catalog.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body>
+  <header class="main-header">
+    <a class="main-header__logo logo">
+      <h1 class="visually-hidden">Сайт Седоны.</h1>
+      <img class="logo__image" src="../images/logo-sedona.svg" width="185" height="96" alt="Логотип Седоны.">
```

Comment:
Правильнее было бы путь указать так:
```suggestion
      <img class="logo__image" src="images/logo-sedona.svg" width="185" height="96" alt="Логотип Седоны.">
```
или так
```suggestion
      <img class="logo__image" src="./images/logo-sedona.svg" width="185" height="96" alt="Логотип Седоны.">
```